test/tools-test.cpp

```diff
@@ -0,0 +1,141 @@
+#include "gtest/gtest.h"
+
+#include "../src/tools.h"
+
+static std::string get_tests_data_dir()
+{
+  return "data";
+}
+
+TEST(CommonTools, getFileSize)
+{
+  std::string fn = get_tests_data_dir() + "/minimal-content/favicon.png";
```

Comment:
Wouldn't be better to create a file in tmp and test it is correctly detected as existing instead of storing a file in the repository ?

Reply:
Probably, no, as this will require extra lines of code. And favicon.png is used anyway in further tests (#143)